src/app/app.component.html

```diff
@@ -1,3 +1,27 @@
 <app-menu></app-menu>
+
+<app-text-image-component
+  [text]="'Meet other students on campus interested in similar projects technologies. ' +
+           'All are welcome, including those with diverse technical backgrounds and different branches.'"
+  [position]="false"
+  [image]="'./assets/home-1.webp'">
+</app-text-image-component>
+
+
+<app-text-image-component
+  [text]="'Learn about a wide range of technical topics where new skills are gained through hands-on workshops, ' +
+          'in-person training and project building activities.'"
+  [position]="true"
+  [image]="'./assets/home-2.webp'">
+</app-text-image-component>
+
+
+<app-text-image-component
+  [text]="'Apply your new learnings and connections to build great solutions for local problems. ' +
+          'Advance your skills, career and network. Give back to your community by helping others learn as well.'"
+  [position]="false"
+  [image]="'./assets/home-3.png'">
+</app-text-image-component>
+
```

Comment:
The nesting is done properly, but the place where you nested them is wrong. We want to nest them in home-page.component.html (/src/app/pages/home-page) because we want our components to be seen only on the home page).